tawk_to.module

```diff
@@ -125,11 +108,11 @@ function tawk_to_page_build(&$page)
         }
     }
 
-    $page['page_bottom']['tawkto_widget'] = array(
+	$variables['page_bottom'] .= array(
```

Comment:
From what I saw in the example from [backdrop admin bar](https://github.com/backdrop/backdrop/commit/6490d6bfd7bf813e71266d128a66cbaf71284a1a#diff-76a37b8cdf08471bdcb862b66c903d2975eda8cf54c52627c21f2dbd06ed83f5R159), I think we can just simply say

```
$variables['page_bottom'] .= tawk_to_render_widget($page_id, $widget_id, $markupOptions);

Reply:
@alvinjohnsonso 
I've done that.  I can see the script at the bottom when I inspect elements:
```
<script type="text/javascript">
    var $_Tawk_API={},$_Tawk_LoadStart=new Date();
    (function(){
    var s1=document.createElement("script"),s0=document.getElementsByTagName("script")[0];
    s1.async=true;
    s1.src="https://embed.tawk.to/60ba37d1de99a4282a1b68ab/1f7bmk35i";
    s1.charset="UTF-8";
    s1.setAttribute("crossorigin","*");
    s0.parentNode.insertBefore(s1,s0);
    })();
    $_Tawk_API.visitor = {
            name  : "administr8or",
            email : "martin@systemhorizons.co.uk",
        };
    var curpage = window.location.href;
    var hash = window.location.hash;
    $_Tawk_API.onLoad = function(){
        if (curpage.indexOf('admin/') != -1 || hash.indexOf('overlay')!= -1) {
            Tawk_API.hideWidget();
        }
    };
    </script>
```
but it doesn't seem to load on the page.

Reply:
Were there any network errors from tawk.to?

Reply:
Yes - there were:
register and https://va.tawk.to/v1/widget-settings?propertyId=60ba37d1de99a4282a1b68ab&widgetId=1f7bmk35i&sv=undefined

However, I tried on incognito and it worked, so guess it doesn't show for the person who set it up?

Reply:
That shouldn't be the case. Could you perhaps post the network request and response for `register`?

Reply:
Ahh but those error aren't caused by the plugin. It's from the widget itself.

Reply:
If we're able to save and remove the widget from the plugin's dashboard and the embed script tag shows on the front page, then that's basically it.

Reply:
Also, if the visibility settings are working properly

Reply:
It was the malwarebytes browser plugin - it is disabled in incognito but active otheriwse.  I trusted va.tawk.to and it was able to load.  Something to be aware of.  I don't know if tawk can get a global truest approval from malwarebytes.

Reply:
Ahh as for the thing I said earlier from the call, I saw somewhere that the URLs instead of having full paths (ex. `some.hostname.com/admin/config/urls/settings`) they had query parameters to access the pages (ex. some.hostname.com?q=admin/config/urls/settings). Which I thought that we might need to change the ajax calls from the `admin.php.inc`.

Then I checked it out and seems like it's a setting that can be enabled/disabled from the backdrop admin page.
![image](https://user-images.githubusercontent.com/12890893/121693323-9cba7700-cafb-11eb-92b5-39c6a908d02c.png)

Reply:
Tested all functions and working.